nautobot/core/views/generic.py

```diff
@@ -38,6 +38,7 @@
     TableConfigForm,
     restrict_form_fields,
 )
+from nautobot.utilities.config import get_settings_or_config
```

Comment:
Alphabetize please - this should be moved up to before the `nautobot.utilities.error_handlers` import.